tests/e2e/scale/noobaa/test_scale_namespace_rpc.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    MCGTest,
+    skipif_ocs_version,
+    vsphere_platform_required,
+    scale,
+)
+from ocs_ci.ocs import constants
+
+logger = logging.getLogger(__name__)
+
+
+@scale
+class TestScaleNamespace(MCGTest):
+    """
+    Test creation of a namespace scale resource
+    """
+
+    @skipif_ocs_version("<4.6")
+    @pytest.mark.parametrize(
+        argnames=["platform"],
+        argvalues=[
+            pytest.param(
+                constants.AWS_PLATFORM, marks=pytest.mark.polarion_id("OCS-2516")
+            ),
+            pytest.param(
+                constants.AZURE_PLATFORM, marks=pytest.mark.polarion_id("OCS-2523")
+            ),
+        ],
+    )
+    def test_scale_namespace_bucket_creation(
+        self, ns_resource_factory, bucket_factory, platform
+    ):
+        """
+        Test namespace bucket creation using the MCG RPC.
+        """
+        # Create the namespace resource and verify health
+        ns_resource_name = ns_resource_factory(platform=platform)[1]
+
+        # Create the namespace buckets on top of the namespace resource
+        bucket_factory(
+            amount=100,
+            interface="mcg-namespace",
+            write_ns_resource=ns_resource_name,
+            read_ns_resources=[ns_resource_name],
+        )
+
+    @skipif_ocs_version("<4.6")
+    @pytest.mark.polarion_id("OCS-2517")
+    @vsphere_platform_required
```

Comment:
what about rhv and bm>

Reply:
Acked.  Change to `on_prem_platform_required` which will include VSPHERE_PLATFORM, BAREMETAL_PLATFORM, BAREMETALPSI_PLATFORM, IBM_POWER_PLATFORM, and RHV_PLATFORM.